packages/camera/camera_avfoundation/ios/Classes/FLTCam.m

```diff
@@ -99,13 +100,15 @@ - (instancetype)initWithCameraName:(NSString *)cameraName
                captureSessionQueue:(dispatch_queue_t)captureSessionQueue
                              error:(NSError **)error {
   return [self initWithCameraName:cameraName
-                 resolutionPreset:resolutionPreset
-                      enableAudio:enableAudio
-                      orientation:orientation
-              videoCaptureSession:[[AVCaptureSession alloc] init]
-              audioCaptureSession:[[AVCaptureSession alloc] init]
-              captureSessionQueue:captureSessionQueue
-                            error:error];
+                   resolutionPreset:resolutionPreset
+                        enableAudio:enableAudio
+                        orientation:orientation
+                videoCaptureSession:[[AVCaptureSession alloc] init]
+                audioCaptureSession:[[AVCaptureSession alloc] init]
+                captureSessionQueue:captureSessionQueue
+                      captureDevice:nil
+      videoDimensionsForFormatBlock:nil
```

Comment:
This should be passing real values, not `nil`; DI behavioral difference should come from the DI itself, not logic in the code.

Reply:
fixed